src/app/(admin)/admin/products/page.tsx

```diff
@@ -1,5 +1,143 @@
-import React from "react";
+"use client";
+
+import { Box, Grid } from "@mui/material";
+import { useForm } from "react-hook-form";
+import { useState, useEffect } from "react";
+import { IProductRow } from "@/components/admin/types";
+import { GridRowSelectionModel } from "@mui/x-data-grid";
+import ProductDataGrid from "@/components/admin/data-grid/ProductDataGrid";
+import NewProductCard from "@/components/admin/product-card/NewProductCard";
+import EditProductCard from "@/components/admin/product-card/EditProductCard";
+import ProductFilterToolbar from "@/components/admin/toolbars/ProductFilterToolbar";
+
+// ----------------------------------------------------------------------
 
 export default function AdminRefreshment() {
-  return <div>AdminRefreshment</div>;
+  const filterMethods = useForm({
+    defaultValues: { search: "", status: "all" },
+  });
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const rows = [
+    {
+      id: 1,
+      category: "Hello",
+      product: "World",
+      status: "inStock",
+      lastUpdated: "30/08/2023 ",
+    },
+    {
+      id: 2,
+      category: "DataGridPro",
+      product: "is Awesome",
+      status: "outStock",
+      lastUpdated: "30/08/2023 ",
+    },
+    {
+      id: 3,
+      category: "MUI",
+      product: "is Amazing",
+      status: "low",
+      lastUpdated: "30/08/2023 ",
+    },
+  ];
+
+  const [filteredRows, setFilteredRows] = useState(rows);
+
+  const [rowSelectionModel, setRowSelectionModel] =
+    useState<GridRowSelectionModel>([]);
+
+  const [isAddCardOpen, setIsAddCardOpen] = useState(false);
+
+  const isOnlyNewProductCardOpen = isAddCardOpen && !rowSelectionModel.length;
+
+  const isOnlyEditProductCardOpen =
+    rowSelectionModel.length > 0 && !isAddCardOpen;
+
+  const isProductCardOpen = rowSelectionModel.length > 0 || isAddCardOpen;
+
+  const { watch } = filterMethods;
+
+  const filterValues = watch();
+
+  const { search, status } = filterValues;
+
+  useEffect(() => {
+    let data = rows;
+
+    if (search) {
+      data = data.filter(
+        (row: IProductRow) =>
+          row.product.toLowerCase().indexOf(search?.toLowerCase()) > -1,
+      );
+    }
+
+    if (status !== "all") {
+      data = data.filter((row: IProductRow) => row.status === status);
+    }
+    setFilteredRows(data);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [search, status]);
+
+  // useEffect(() => {
```

Comment:
are these going to be used in the future?

Reply:
yep, once the api is ready.